frontend/.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     "react/no-array-index-key": "off",
     "jsx-a11y/click-events-have-key-events": "off",
     "jsx-a11y/no-static-element-interactions": "off",
+    "@typescript-eslint/no-useless-constructor": "off",
```

Comment:
What does this do?

Reply:
There's one file that has a constructor eslint deems useless/unnecessary. I don't remember which file exactly, but I'm pretty sure it was in node_modules somewhere... so not even our own code. I'll check again to see if there's any other workaround!

Reply:
Woah I just ran it without that line and it compiled! guess I changed some package between now and when I had that error and accidentally fixed it lol 😅
thanks for catching this!!